x-pack/test/functional/services/ml/add_trained_models_flyout.ts

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+
+import type { FtrProviderContext } from '../../ftr_provider_context';
+
+type FlyoutTabManualDownload = 'Manual Download';
+type FlyoutTabClickToDownload = 'Click to Download';
+export type FlyoutTabs =
+  | [FlyoutTabManualDownload]
+  | [FlyoutTabClickToDownload, FlyoutTabManualDownload];
+
+export function TrainedModelsFlyoutProvider({ getService }: FtrProviderContext) {
+  const testSubjects = getService('testSubjects');
+  const retry = getService('retry');
+
+  const tenSeconds = 10000;
```

Comment:
have you tried with 5 seconds? it's a timeout we use in most places

Reply:
I'll try that next.